src/hooks/useGetToolsQuery.ts

```diff
@@ -1,7 +1,5 @@
 import { toolsApi } from "../services/refact/tools";
-import { useHasCaps } from "./useHasCaps";
 
 export const useGetToolsQuery = () => {
-  const hasCaps = useHasCaps();
-  return toolsApi.useGetToolsQuery(undefined, { skip: !hasCaps });
```

Comment:
If I remember correctly, available tools relies on caps being available from the lsp.

So this would be executed before caps are available if the "send_imediantly" flag was set by the ide, this happens when using the tool box or right click in vscode.